src/Api/Client.php

```diff
@@ -24,13 +24,13 @@ public function getProgramInfo(string $programId): ProgramInfo
         return new ProgramInfo($response->json('program.id'), $response->json('program.name'));
     }
 
-    public function getProgramMembers(string $programId): array
+    public function getProgramMembers(string $programId, $params = []): array
```

Comment:
`$params` is too vague... what params? what params are available to me? if we're going to use this for pagination, let's create a `PaginationParams` type (maybe with a better name and accept that as an additional arg)